### ISSUE TITLE
Add support for DaVinci Resolve Studio as well?

### ISSUE THREAD

**Issue by ryanmoon** [autopkg/timsutton-recipes#33]
I have to support both, would you be open to a pull request for adding Studio?

**Comment by timsutton** [comment]
Certainly. We have Studio here as well, but it's not something I've needed to deploy via Munki. Tangentially related, do you have a reliable way to differentiate between the two installers, and/or what they install?

**Comment by ryanmoon** [comment]
I currently do not have anything short of making an MD5 sum. They went the route of naming the application the same exact way. Which baffles me to no end. 

Ideally I'd want a way to install both, and then if someone needs to use the Studio version they could just check out a dongle, or we'd use our dongle server to make it available to the machine. However I don't think that's going to work for the time being. 

But that being said, I think I'll try throwing Composer at an install of each and see if I can find anything different.

**Comment by timsutton** [comment]
Yes, a snapshot comparison would be interesting. It seems entirely possible to me that a "studio" installation is nothing more than a single configuration tweak.

**Comment by timsutton** [comment]
If I open up the two packages, I can see at the very least that `/Applications/DaVinci Resolve/DaVinci Resolve.app/Contents/Frameworks/CG.framework/Cg` is a significantly larger binary in the Studio edition.

**Comment by timsutton** [comment]
And so does the main app binary:

<img width="1584" alt="screenshot 2017-03-08 14 37 30" src="https://cloud.githubusercontent.com/assets/119358/23720492/caf6d6e6-040c-11e7-8a76-32a8749a7fec.png">

**Comment by ryanmoon** [comment]
So I've found that the binary, located at `/Applications/DaVinci\ Resolve/DaVinci\ Resolve.app/Contents/MacOS/Resolve` is different between Studio and non Studio. However if you replace the binary from DaVinci, with the one from DaVinci Studio, then launch the app, you get the Studio splash screen and are immediately asked for a dongle. Insert dongle and then you get errors for some of the dylibs.
